rtrlib/rtr_mgr.h

```diff
@@ -67,28 +68,37 @@ struct rtr_mgr_group {
 	enum rtr_mgr_status status;
 };
 
+//TODO Find a nicer way todo a linked list (without writing our own)
+struct rtr_mgr_group_node {
```

Comment:
The struct is also used in the `test_dynamic_groups` test. So we keep it in the header file?